debug_toolbar/compat.py

```diff
@@ -0,0 +1,77 @@
+"""
+This file exists to contain all Django and Python compatibility issues.
+
+In order to avoid circular references, nothing should be imported from
+debug_toolbar.
+"""
+
+from django.conf import settings
+
+try:
+    from django.core.cache import CacheHandler, caches
+except ImportError:  # Django < 1.7
```

Comment:
You've used `< Django 1.7` everywhere else (but I prefer `Django < 1.7`).

Just normalize it one way or another.